### DISCUSSION TITLE
Dark Mode?

### DISCUSSION THREAD

**Discussion by raster** [pixelfed/pixelfed-rn#34] [Bug Reports]
Is Dark Mode gone? I know it had some issues but I still preferred using it.

**Comment by SiskoUrso** [comment]
Also was the first thing I went looking to turn on.

**Comment by pguimier** [comment]
Same here. Now the 1.0.0.45 follows the daytime theme, but tjis would be nice to choose : 
- light
- dark
- follow the system

**Comment by aplsms** [comment]
Dark mode is gone. Does any body know pixelfed replacement with dark mode?

**Comment by ketelaarsmaarten** [comment]
Dark mode is not a wish but a **must** due to photophobia, I hope this can be solved soon.

**Comment by InsaneAwesomeTony** [comment]
My eyes are permanently burned due to the white screen. o7 best quality of life feature.

**Comment by ltning** [comment]
I'll add my voice here. Dark mode is kinda important this day and age. It's a matter of preference, of course, but the lack of dark mode is the primary reason I'm not using the app much (despite running a Pixelfed instance).

To be clear - other than that the app is really looking good.

**Comment by Hellmark** [comment]
For those of us with disabilities, dark mode can be more than just a preference. It is literally painful for me to see large amounts of white on most displays.
